src/platform/common.c

```diff
@@ -40,3 +49,20 @@ void IntrMain(void)
 
 #undef CHECK_INTR
 }
+
+#define DO_UNSAFE_CONVERT
+#include "../tools/gbagfx/global.h"
```

Comment:
Wow lol, would it not just be best to copy these functions in? Including c files directly from some library feels a bit wrong.

Reply:
I feel it's better, since it's the original source and not a copy of it.
The #ifdefs are a bit ugly, sure, but I basically see it as using it as a "Single-Header" library.